core/src/mothra_api/api.rs

```diff
@@ -176,42 +176,45 @@ pub fn config(args: Vec<String>) -> ArgMatches<'static> {
     .arg(
         Arg::with_name("listen-address")
             .long("listen-address")
-            .value_name("Address")
-            .help("The address artemis will listen for UDP and TCP connections. (default 127.0.0.1).")
+            .value_name("ADDRESS")
+            .help("The address the client will listen for UDP and TCP connections. (default 127.0.0.1).")
+            .default_value("127.0.0.1")
+            .takes_value(true),
+    )
+    .arg(
+        Arg::with_name("port")
+            .long("port")
+            .value_name("PORT")
+            .help("The TCP/UDP port to listen on. The UDP port can be modified by the --discovery-port flag.")
             .takes_value(true),
     )
     .arg(
         Arg::with_name("maxpeers")
             .long("maxpeers")
             .help("The maximum number of peers (default 10).")
+            .default_value("10")
             .takes_value(true),
     )
     .arg(
         Arg::with_name("boot-nodes")
             .long("boot-nodes")
             .allow_hyphen_values(true)
-            .value_name("BOOTNODES")
+            .value_name("ENR-LIST")
             .help("One or more comma-delimited base64-encoded ENR's to bootstrap the p2p network.")
             .takes_value(true),
     )
-    .arg(
-        Arg::with_name("port")
-            .long("port")
-            .value_name("Artemis Port")
-            .help("The TCP/UDP port to listen on. The UDP port can be modified by the --discovery-port flag.")
-            .takes_value(true),
-    )
     .arg(
         Arg::with_name("discovery-port")
             .long("disc-port")
-            .value_name("DiscoveryPort")
+            .value_name("PORT")
```

Comment:
Is this going to conflict with the (normal) PORT defined above?

Reply:
It's purely cosmetic.  In reality, it's totally useless for mothra.